CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # App Center SDK for React Native Change Log
 
+## Version 2.5.0 (Under active development)
```

Comment:
To reduce merge conflict with RN September release, I am adding this to changelog for the next month's release.